spring-data-mongodb/src/test/java/org/springframework/data/mongodb/config/GeoJsonJackson3ConfigurationIntegrationTests.java

```diff
@@ -22,27 +22,30 @@
 
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.context.annotation.Configuration;
+import org.springframework.data.mongodb.core.geo.GeoJsonJackson3Module;
 import org.springframework.data.mongodb.core.geo.GeoJsonModule;
 import org.springframework.data.web.config.EnableSpringDataWebSupport;
 import org.springframework.test.context.ContextConfiguration;
 import org.springframework.test.context.junit4.SpringRunner;
 
 /**
- * Integration tests for {@link GeoJsonConfiguration}.
+ * Integration tests for {@link GeoJsonJackson3Configuration}.
  *
- * @author Oliver Gierke
+ * @author Bjorn Harvold
+ * @author Jens Schauder
  */
 @RunWith(SpringRunner.class)
 @ContextConfiguration
-public class GeoJsonConfigurationIntegrationTests {
+public class GeoJsonJackson3ConfigurationIntegrationTests {
 
 	@Configuration
 	@EnableSpringDataWebSupport
 	static class Config {}
 
-	@Autowired GeoJsonModule geoJsonModule;
+	@Autowired
+	GeoJsonJackson3Module geoJsonModule;
 
-	@Test // DATAMONGO-1181
+	@Test // GH-5100
 	public void picksUpGeoJsonModuleConfigurationByDefault() {
 		assertThat(geoJsonModule).isNotNull();
 	}
```

Comment:
Please do not remove the the wiring of the old module since it is still present and should be picked up in case Jackson2 is present. `SpringDataWebConfigurationImportSelector` will pick and choose based on what's present.
It should be possible to wire both, right?